contracts/modules/src/staking.rs

```diff
@@ -19,12 +21,24 @@ pub trait StakingModule {
         slash_quorum: usize,
         user_whitelist: &ManagedVec<ManagedAddress>,
     ) {
-        let nr_board_members = user_whitelist.len();
+        for user in user_whitelist {
+            let _ = self.user_whitelist().insert(user);
+        }
+
+        let nr_board_members = self.user_whitelist().len();
         require!(nr_board_members > 0, "No board members");
         require!(
-            slash_quorum <= nr_board_members,
+            nr_board_members <= MAXIMUM_BOARD_MEMBERS,
```

Comment:
Should we also have this check that `user_whitelist().len() <= MAXIMUM_BOARD_MEMBERS` in in `add_board_member`?